docs/source/fx.rst

```diff
@@ -293,10 +293,188 @@ Writing Transformations
 
 TODO
 
-Debugging Transformations
--------------------------
+Debugging
+-----------
+
+Introduction
+^^^^^^^^^^^^^^^^
+
+Once we've generated Python code from our symbolically-traced Module,
+we'll want to verify that the proper semantics were preserved. If they
```

Comment:
I think we want to emphasize debugging transformations here. Debugging issues with symbolic tracing should (hopefully) be rare for external users